root/entity/Details.js

```diff
@@ -89,10 +91,23 @@ const Details = ({
         <tr>
           <th>{l('XML:')}</th>
           <td>
-            <XMLLink
+            <WSLink
               entityGid={entity.gid}
               entityProperties={entityProperties}
               entityType={entityType}
+              isJson={false}
```

Comment:
Probably unneeded? Or if the idea is to highlight this is XML, maybe replace `isJson` with `format` and use `format={'xml'}` instead here?

Reply:
IIRC Flow complained without it? Can check when I get home